Sketchify your Photo/README.md

```diff
@@ -0,0 +1,81 @@
+# Sketchify your Photo
+
+### Ever wondered how the photo editing apps generate a automatic pencil sketch of your photo? Here is a micro-byte wherein you can Sketchify your Photo using Open CV.
```

Comment:
`generate an automatic pencil sketch`

I dunno what's with me and grammar. :sweat_smile: